web/packages/design/src/SyncStamp/SyncStamp.tsx

```diff
@@ -22,16 +22,18 @@ import Flex from 'design/Flex';
 import { SyncAlt } from 'design/Icon';
 import { P3 } from 'design/Text';
 
-export function SyncStamp({ date }: { date: Date }) {
+export function SyncStamp({ date }: { date?: Date }) {
+  let content = 'N/A';
+  if (date && date.getTime() >= 0) {
+    content = formatDistanceStrict(new Date(date), new Date(), {
```

Comment:
`date` is already of type `Date`; plus we can use a different utility function:
```suggestion
    content = formatDistanceToNowStrict(date, {
```